### ISSUE TITLE
twisted/test/test_pb.py should use setUp instead of setUpClass and not have any tear down

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2744]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2744|
|Type|task|
|Created|2007-07-20 14:40:47Z|

PagingTestCase writes a file in setUpClass and deletes it tearDownClass.

The file should be written in a setUp, instead.  Nothing measurable is gained by doing this only once.  The file shouldn't be deleted, either, in case it is needed for debugging, so the tearDownClass should be removed and nothing added in its place.

<details><summary>Searchable metadata</summary>

```
trac-id__2744 2744
type__task task
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1184942447000000 1184942447000000
changetime__1185626525000000 1185626525000000
version__None None
owner__ 
cc__therve
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun removed owner|
|-|-|

Done in setup-teardown-pb-2744

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set owner to @exarkun|
|-|-|

Please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

(In [20856]) Merge setup-teardown-pb-2744

Author: exarkun
Reviewer: therve
Fixes [#2744](https://github.com/twisted/twisted/issues/2744)

Replace setUpClass on PagingTestCase with a setUp method and remove
the tearDownClass which deleted the file created by setup.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
